Objective-C.gitignore

```diff
@@ -28,6 +32,12 @@ xcuserdata/
 *.dSYM.zip
 *.dSYM
 
+## Finder
+.DS_Store
```

Comment:
This rule is already managed by `Global/macOS.gitignore`. Please drop it from this PR.